ccdproc/ccddata.py

```diff
@@ -20,49 +20,60 @@
 
 
 class CCDData(NDDataArray):
-
-    """A class describing basic CCD data
+    """A class describing basic CCD data.
 
     The CCDData class is based on the NDData object and includes a data array,
     uncertainty frame, mask frame, meta data, units, and WCS information for a
     single CCD image.
 
     Parameters
     -----------
-    data : `~numpy.ndarray` or :class:`~ccdproc.CCDData`
+    data : `~ccdproc.CCDData`-like or `numpy.ndarray`-like
         The actual data contained in this `~ccdproc.CCDData` object.
         Note that this will always be copies by *reference* , so you should
-        make copy the ``data`` before passing it in if that's the  desired
+        make copy the ``data`` before passing it in if that's the desired
```

Comment:
As long as we are cleaning up, could you please change `make copy` to `make a copy`?